.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -376,8 +378,8 @@ jobs:
       if: steps.rust-tools-cache.outputs.cache-hit != 'true'
```

Comment:
you'll need to update the cache line above to "break" the cache, it should become

```
key: ${{ matrix.os }}_sccache-0.7.7_mdbook-0.4.36_mdbook-linkcheck-0.7.7
```